src/ekobox/info_file.py

```diff
@@ -22,8 +22,8 @@ def create_info_file(theory_card, operators_card, num_members, info_update):
 
     Returns
     -------
-        : dict
-        info file in lhapdf format
+        dict :
```

Comment:
Since you're not specifying a name, you don't need a colon at all, neither before nor after.

Please, follow the reference https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html